### DISCUSSION TITLE
SDI Installation RHEL/Centos 8

### DISCUSSION THREAD

**Discussion by michallawson** [HCL-TECH-SOFTWARE/connections-automation#41] [General]
Installation of SDI failed for me when installing onto a Centos 8.3 server.

Error message was: msg": "Failed to validate GPG signature for compat-libstdc++-33-3.2.3-72.el7.x86_64"

Not sure what caused this, but found article on IBM which may be relevant:

https://www.ibm.com/support/pages/sdi-72-how-install-red-hat-enterprise-linux%C2%AE-8

**Comment by dejo1307** [comment]
Good catch, we never tested yet past CentOS 8.2. Will add it to our list for this and next week. Will check this specific thing tomorrow if this is a fast fix just to unblock here.

**Reply by michallawson** [reply]
Thank you. I followed the instructions given by IBM and amended the ansible scripts to skip past that section and seems to have worked, Now fails on download and upgrade TDI.. with following message:  

_"msg": "The conditional check '__tdi_upgrade | bool' failed. The error was: error while evaluating conditional (__tdi_upgrade | bool): '__tdi_upgrade' is undefined\n\nThe error appears to be in '/etc/ansible/roles/third_party/ibm/tdi-install/tasks/main.yml': line 6, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name:                       Download and upgrade TDI\n  ^ here\n""_

Suspect I am missing a filename somewhere.

Regards

Michael

**Reply by dejo1307** [reply]
I think you by mistake removed in vars one underscore from _tdi_upgrade' is undefined (note that in error it complains that _tdi_upgrade' is undefined doesn't exist, which doesn't, what exist is __tdi_upgrade' is undefined (two underscores in front)

**Reply by michallawson** [reply]
Thank you and will check.
